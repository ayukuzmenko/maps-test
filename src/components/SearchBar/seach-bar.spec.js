import React from './node_modules/react';
import { shallow } from './node_modules/enzyme';
import { shallowToJson } from './node_modules/enzyme-to-json';
import { SearchBar } from '.';

window.ymaps = {
  ready: jest.fn(),
};

describe('Serch bar component test', () => {
  it('Should match to snapshot', () => {
    const searchBarContainer = shallow(<SearchBar />);

    expect(shallowToJson(searchBarContainer)).toMatchSnapshot();
  });
});
