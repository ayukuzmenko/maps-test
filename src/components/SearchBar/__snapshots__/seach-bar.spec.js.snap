// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Serch bar component test Should match to snapshot 1`] = `
<div>
  <p
    className="header"
  >
    Введите адрес точки:
  </p>
  <input
    className="suggest"
    id="suggest"
    name="address"
    type="text"
  />
</div>
`;
